bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -36,14 +50,18 @@
       <div class="content-wrapper">
         <div class="phone">
           <div class="fxos-cta">
-            <h1><span>{{_('Firefox OS')}}</span></h1>
+            <h1>{{ high_res_img('img/firefox/os-new/fxos-wordmark.png', {'alt': _('Firefox OS'), 'width': '420', 'height': '85'}) }}</h1>
```

Comment:
Did you try SVG for the wordmark? If PNG is the smaller file then never mind (PNG is also better supported in old browsers, so there's that).